modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/service/impl/CTCollectionLocalServiceImpl.java

```diff
@@ -107,7 +109,7 @@ public CTCollection deleteCTCollection(CTCollection ctCollection)
 				continue;
 			}
 
-			ctEntryLocalService.deleteCTEntry(ctEntry);
+			ctEntryPersistence.remove(ctEntry);
```

Comment:
@lanceji92674957 @Preston-Crary I don't think this kind of change is safe. The service method has Indexable annotation on it, simply switch to persistence will lose the indexing.